Cargo.toml

```diff
@@ -89,88 +89,88 @@ ariadne = "0.6"
 assert_fs = "1"
 assert_cmd = "2"
 assert-json-diff = "2"
-async-trait = "0.1.83"
+async-trait = "0.1"
 backtrace = "0.3"
-backon = { version = "1.4.0", features = ["std", "tokio-sleep"] }
+backon = { version = "1.4", features = ["std", "tokio-sleep"] }
 base64 = "0.22"
 billboard = "0.2"
-buildstructor = "0.6.0"
-bytes = "1.8.0"
+buildstructor = "0.6"
+bytes = "1"
 cargo_metadata = "0.23"
 calm_io = "0.1"
 camino = { version = "1", features = ["serde1"] }
 clap = "4"
 chrono = "0.4"
 ci_info = "0.14"
-comfy-table = { version = "7.1.4", features = ["custom_styling"] }
+comfy-table = { version = "7", features = ["custom_styling"] }
 console = "0.16"
-derive-getters = "0.5.0"
+derive-getters = "0.5"
 dialoguer = { version = "0.12", features = ["fuzzy-select"] }
-directories-next = "2.0"
+directories-next = "2"
 dotenvy = "0.15"
 flate2 = "1"
 futures = "0.3"
-git-url-parse = "0.6.0"
+git-url-parse = "0.6"
 git2 = { version = "0.20", default-features = false }
 graphql_client = "0.14"
 heck = "0.5"
-humantime = "2.1.0"
-http = "1.1.0"
-http-body = "1.0.1"
-http-body-util = "0.1.2"
+humantime = "2"
+http = "1"
+http-body = "1"
+http-body-util = "0.1"
 httpmock = "0.8"
-hyper = "1.0"
+hyper = "1"
 indoc = "2"
-itertools = "0.14.0"
+itertools = "0.14"
 lazycell = "1"
-lazy_static = "1.4"
+lazy_static = "1"
```

Comment:
Is there any value in keeping the minor version when there's a non-0 major version, instead of only specifying the major version? This seems like it's losing useful minor version information.

Reply:
Sure, but once the major version is 1 or greater, we still might want to constrain the minor version too? So `lazy_static = "1.4"` was maybe already the right level of specificity here?

What do we gain by allowing any 1.x.y version here? Do we need to test new versions that are now allowed?